notebooks/tinybase/Basic.ipynb

```diff
@@ -0,0 +1,1183 @@
+   "id": "6c1f5af7-f5e9-41d9-a849-bab0ebc7dd9f",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "[]"
+      ]
+     },
+     "execution_count": 8,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "f.input.args"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "id": "e0afb76d-d1b7-4b42-925f-fb117d58025e",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "{}"
+      ]
+     },
+     "execution_count": 9,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "f.input.kwargs"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "id": "6a5c3235-9c6b-481f-b316-db7420d1ad43",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "f.input.kwargs['n'] = 10"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "id": "4ade8d6a-6ce2-4f3a-b43d-71e1f87125bf",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "{'n': 10}"
+      ]
+     },
+     "execution_count": 11,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "f.input.kwargs"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "id": "a39cfb50-4ed6-49ab-8ffb-af236cf61153",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "exe = f.run()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "id": "aa095e57-3d3c-4af5-9a94-eda6af34e2b3",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "{'status': (ReturnStatus(Code.DONE, None),),\n",
+       " 'output': (<pyiron_contrib.tinybase.node.FunctionOutput at 0x7fa49a99ec90>,)}"
+      ]
+     },
+     "execution_count": 13,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "exe._run_machine._data"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "id": "82d0ca07-12c2-4cd9-bba3-1f210a907b41",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "144"
+      ]
+     },
+     "execution_count": 14,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "exe.output[0].result"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "id": "24994f4a-d5cd-4aad-857f-a33ddd0eaf23",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "(1.0009997060114983, 9.59599856287241e-06)"
+      ]
+     },
+     "execution_count": 15,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "exe._run_time, exe._collect_time"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a6cbcd0e-2a04-4e93-add5-0f423fc1fa69",
+   "metadata": {},
+   "source": [
+    "## We don't actually have to use an executor if we just want a result"
```

Comment:
This example is difficult to understand, because it doesn't tell how to make a comparable output as in the subsection above.